src/Jobs/Recruit.Vacancies.Jobs/appSettings.Development.json

```diff
@@ -27,7 +27,7 @@
     "VacancyAnalyticEventsSqlDbConnectionString": "Data Source=127.0.0.1,1401;Initial Catalog=das-vacancy-analytic-events;User Id=sa;Password=Ozzyscottypassword1;MultipleActiveResultSets=True"
   },
   "Slack": {
-    "WebHookUrl": ""
+    "FindAnApprenticeshipDetailPrefixUrl": "https://www.findapprenticeship.service.gov.uk/apprenticeship/"
```

Comment:
Wonder if this should just contain the domain name only `https://www.findapprenticeship.service.gov.uk`. Then the notifier can append the path as needed.  Then we could redirect to different locations in FAA for different types of notifications without needed extra config. But there is no demand for this yet so this is fine. Just a musing.

Reply:
This is for purpose so it is fine.